CMakeLists.txt

```diff
@@ -129,6 +129,7 @@ option(CUDAQ_TEST_MOCK_SERVERS "Enable Remote QPU Tests via Mock Servers." OFF)
 option(CUDAQ_DISABLE_RUNTIME "Build without the CUDA Quantum runtime, just the compiler toolchain." OFF)
 option(CUDAQ_SKIP_MPI "Do not build with MPI, even when it is available." OFF)
 option(CUDAQ_DISABLE_CPP_FRONTEND "Build without the CUDA Quantum C++ Clang-based Frontend." OFF)
+option(CUDAQ_ENABLE_CC "Enable CUDA Quantum code coverage generation." OFF)
```

Comment:
Can we use something other than `CC` for code coverage?

(CC is already overloaded with it being an environment variable, a dialect, and a C compiler.)